tests/unit/statement.c

```diff
@@ -80,16 +80,16 @@ int main(int argc, char *argv[]) {
              drizzle_error(con));
 
   ret = drizzle_select_db(con, "test_stmt");
-  ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "use test_stmt");
+  ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "USE test_stmt");
```

Comment:
But shouldn't this be correct, the **ASSERT_EQ** is after the call to `drizzle_select_db`?